docs/usage.md

```diff
@@ -211,7 +217,37 @@ Notes:
 - If `--additional_fasta` is provided then the features in this file (e.g. ERCC spike-ins) will be automatically concatenated onto both the reference FASTA file as well as the GTF annotation before building the appropriate indices.
 - When using `--aligner star_rsem`, both the STAR and RSEM indices should be present in the path specified by `--rsem_index` (see [#568](https://github.com/nf-core/rnaseq/issues/568)).
 
-#### Indices
+#### Reference genome
+
+It is recommended to provide the most complete reference genome for your species, without additional loci (haplotypes) or patches. For model organisms such as mouse or human, this is the "primary assembly," which includes the reference chromosomes and some additional scaffolds. For the human assembly GRCh38 (hg38), use the `GRCh38.primary_assembly.genome.fa.gz` file from GENCODE or the `Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz` file from Ensembl. These files cover the largest portion of the reference genome without including multiple copies of the same sequence, which can confuse aligners like STAR.
+
+Most other species (e.g., fly, cow, dog) do not have a primary assembly. In these cases, use the complete reference sequence, or "toplevel" assembly. The main difference between the primary and toplevel assemblies is the inclusion of alternative loci (haplotypes), which typically do not exist for species outside of mouse and human.
```

Comment:
I am hesitant recommending the top level assemblies unaltered. For the purpose of RNA-seq, that will blow up the number of multi-mapping reads and ultimately affect the quantification. Do you happen to have a reference how big the impact will be? My gut feeling would rather suggest using e.g. [seqkit grep](https://bioinf.shenwei.me/seqkit/usage/#grep) to remove the alternate loci myself if no primary assemblies are available.

Reply:
I haven't really worked with species outside mouse/human so I'm not really sure. This suggestion came from @tdanhorn [here](https://github.com/nf-core/rnaseq/issues/1086#issuecomment-2156321533). Maybe they/someone could comment on if this is correct for other species?

Reply:
Well, the way how I understand it is that Thomas Danhorn only explains the difference between a _primary_ and a `toplevel `assembly. Everything that is said there is absolutely accurate. 

I just question that we should recommend using the toplevel assembly straightforwardly in case a published primary assembly is lacking.

Reply:
Ah, I got the impression that if there was no primary assembly then toplevel probably didn't contain alternative loci etc. so it would be fine to use. Maybe that's not correct though. Happy to have it changed to whatever is most useful.